release-tools/do-copyright-year

```diff
@@ -54,5 +54,5 @@ git diff-tree -r --name-status `git rev-list -1 --before=$NYD HEAD`..HEAD | \
     )
 echo Files changed: $(git status --porcelain | grep '^ *M' | wc -l)
 echo Committing change locally.
-git commit -m 'Update copyright year'
+git commit -m $'Update copyright year\n\nRelease: yes'
```

Comment:
Not sure that this is necessary at this point, since trailers are added with `addrev` on approval anyway...

Reply:
I guess it isn't, though it can't hurt. Removing it would assume addrev is always used after using the release scripts. I guess we don't need to assume that.